doc/grattexDocumentation.Rnw

```diff
@@ -833,17 +833,76 @@ It may have mandatory or optional arguments, which occur immediately after \verb
+      \item Save the Word document without spaces, \eg~\verb|mapping_converted.docx|.
+    \end{enumerate}
+
+  \subsection{Running \texttt{pandoc}}
+      Next, convert the \texttt{.docx} to a \texttt{.tex} file using \texttt{pandoc}. Open \texttt{terminal} (Mac) or \texttt{cmd} (Windows) and set the working directory to the location of your \texttt{.docx} document by:
+      
+         \begin{lstlisting}
+      cd @@<path to folder containing document>@@
+         \end{lstlisting}
+      \eg 
+         \begin{lstlisting}
+      cd /Users/mackeyw/Documents/convert
+         \end{lstlisting}
+
+      Now that you have told \texttt{terminal}/\texttt{cmd} where to look, you can run \texttt{pandoc} using the following command. Replace \texttt{report} with the title of your document.
+      \begin{lstlisting}
+      pandoc -s @@<name>@@.docx -s --wrap=none --top-level-division=chapter 
+      -o @@<name>@@.tex
+      \end{lstlisting}
+    
+      \eg 
+      \begin{lstlisting}[language=bash]
+      pandoc -s mapping_converted.docx -s --wrap=none --top-level-division=chapter -o mapping_converted.tex
+      \end{lstlisting}
+
+    This will create a \texttt{.tex} file in your folder. 
+
+  \subsection{Initial compile of \texttt{.tex} file}
+
+    The final step is to get the \texttt{.tex} file to compile before you bring it into ShareLaTeX. This involves `commenting out' a few things that \texttt{pandoc} can't do well (\eg~tables) and some that we haven't given it the information to do at all (\eg~figures/charts).
+    
+    \begin{enumerate}
+        \item   Open the \texttt{.tex} in your local \TeX~program. 
+        \item   Using find-replace, search \verb|\includegraphics| and replace with \verb|%%\includegraphics|. This will comment out all figures in the document.
+        \item   Using find, search \verb|\begin{longtable}| and comment out everything between and including \verb|\begin{longtable}| and \verb|\end{longtable}|. Note that boxes will have been converted to \verb|longtable| environments, which also need to be commented out.
+        \item   \texttt{pandoc} will sometimes add an unnecessary \verb|\hypertarget{}| around chapter/section names. You can manually remove these, or replace using \texttt{regex}:
+          \begin{itemize}
+                \item Find:     \verb|\\hypertarget{((\w*.*)[\r\n](\w*.*))}|
+                \item Replace:  \verb|$3|
+          \end{itemize}
+  \end{enumerate}
+
+After making these changes, compile your document. Correct any errors preventing the compile (if there are any). When it compiles---yay---you can move the \texttt{.tex} code into your Share\LaTeX~document, starting with the text after \verb|\begin{document}|, ignoring the preamble (see following subsection).
+
+From here: 
+\begin{enumerate}
+  \item   Add your graphics (see \vref{sec:Figures}})
+  \item   Add your references (see \vref{chap:citations-and-bibliography})
+  \item   Add your tables (see \vref{sec:tables})
+\end{enumerate}
+  
```

Comment:
Not sure if it's worth including as an alternative, using `grattanReporter::word2tex()`?

Reply:
Just testing it out. Works well after some playing around; and R is probably more familiar to people than terminal/cmd. It could replace all of 2.5.2, and 2.5.3 point 2. 

Could also add to it to remove the \hypertarget?